#goal definition
geometry_msgs/PoseStamped target_pos
---
#result definition
bool arrived
---
#feedback
geometry_msgs/Pose actual_pos
